domains/sizrox.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hoodnabeill",
+        "email": "[hoodnabeil@gmail.com]"
```

Comment:
This is not the proper way for the email field, it should just be the email not surrounding by anything.